### PR TITLE
Fix issue #59

### PR DESCRIPTION
**Pull request by mattock**
This fixes the four original issues identified in https://github.com/OpenVPN/openvpn-build/issues/59

### CODE REVIEW

**Comment by selvanair** [comment]
This looks good. The last change will become redundant when we make userspace always installed, but still good to have for commandline usage. ACK.

**Comment by mattock** [comment]
Let's not merge this quite yet. There is some syntax error in the new messagebox, even though I checked the NSIS reference:
```
!insertmacro: _If
!insertmacro: end of _If
Usage: MessageBox mode messagebox_text [/SD return] [return_check label_to_goto_if_equal [return_check2 label2]]
    mode=modeflag[|modeflag[|modeflag[...]]]
    modeflag=(MB_ABORTRETRYIGNORE|MB_OK|MB_OKCANCEL|MB_RETRYCANCEL|MB_YESNO|MB_YESNOCANCEL|MB_ICONEXCLAMATION|MB_ICONINFORMATION|MB_ICONQUESTION|MB_ICONSTOP|MB_USERICON|MB_TOPMOST|MB_SETFOREGROUND|MB_RIGHT
Error in script "openvpn.nsi" on line 511 -- aborting creation process
FATAL: makensis
FATAL: pack installer >&2


The command "$SCRIPT" exited with 1.

Done. Your build exited with 1.
```

**Comment by mattock** [comment]
There is one IDNO missing from front of /SD in the modified messagebox.

File: windows-nsis/openvpn.nsi [hunk 1]
```diff
@@ -508,13 +508,12 @@ Function .onInit
 
 ${IfNot} ${AtLeastWinVista}
 
-	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES IDNO guiEndNo
-	Goto guiEndYes
+	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES DownloadForWinXP IDNO DontDownloadForWinXP
```
Comment (selvanair) [inline windows-nsis/openvpn.nsi]: I see, it  has to be  /SD IDYES IDYES jump_if_IDYES IDNO jump_if_IDNO
@chipitsine 's suggestion in PR61 may be used here (or merge that one first) PR 61 also changes the /SD IDYES to /SD IDNO which is better for silent install.

**Comment by selvanair** [comment]
Sorry, I didn't try building it earlier -- the changes looked so innocuous, but never take things for granted... Yes, now it builds and installer works fine on Windows 7 -- all https links work too. No XP here, so can't test that -- the message box branching looks right though. The change to /SD IDNO from /SD IDYES is good too.

**Comment by mattock** [comment]
@selvanair : ok, good. I will squash the two messagebox commits on Monday and then merge this.
@chipitsine : thanks for the Travis work - it saved us this time :+1:

**Pull request merged**